project/app/views.py

```diff
@@ -0,0 +1,13 @@
+from django.http import JsonResponse, HttpResponse
```

Comment:
Import should be on separate lines: https://github.com/geoadmin/doc-guidelines/blob/master/PYTHON.md#5-imports-formatting